tableauserverclient/server/request_options_base.py

```diff
@@ -0,0 +1,9 @@
+import abc
+
+
+class RequestOptionsBase(object):
```

Comment:
Ok, I wanted to take advantage of the builtin abstract class support. But it's kind of an overkill for this simple use case and like you said, probably makes the code more difficult to understand as well. will switch back to simple inheritance =)